src/linker/runner.py

```diff
@@ -45,10 +45,11 @@ def main(
     argv = [
         "--snakefile",
         str(snakefile),
-        "--jobs=2",
-        "--latency-wait=10",
         "--cores",
-        "1",
+        "all",
```

Comment:
I was a little confused about this - why are you using all cores and unlimited jobs? What is this actually doing?

Reply:
See discussion here--https://github.com/ihmeuw/linker/pull/84#discussion_r1560091061
This allows by default to use all the cores available on the host machine (locally) to run rules, and to submit however many simultaneous jobs to the cluster scheduler.